pages/projects.js

```diff
@@ -0,0 +1,357 @@
+    'Thursday, August 14th 2021',
+    'Thursday, August 14th 2021'
+  ),
+  createData(
+    3,
+    'Logout screen',
+    true,
+    'Abid, Emily, Blake and Dan',
+    'Thursday, August 14th 2021',
+    'Thursday, August 14th 2021'
+  ),
+  createData(
+    4,
+    'Image screen',
+    false,
+    'Abid, Emily, Blake and Dan',
+    'Thursday, August 14th 2021',
+    'Thursday, August 14th 2021'
+  ),
+];
+
+function descendingComparator(a, b, orderBy) {
+  if (b[orderBy] < a[orderBy]) return -1;
+  if (b[orderBy] > a[orderBy]) return 1;
+  return 0;
+}
+
+function getComparator(order, orderBy) {
+  return order === 'desc'
+    ? (a, b) => descendingComparator(a, b, orderBy)
+    : (a, b) => -descendingComparator(a, b, orderBy);
+}
+
+const headCells = [
+  {
+    id: 'name',
+    numeric: false,
+    disablePadding: true,
+    label: 'Project',
+  },
+  {
+    id: 'shared',
+    numeric: false,
+    disablePadding: false,
+    label: 'Shared',
+  },
+  {
+    id: 'teamMembers',
+    numeric: false,
+    disablePadding: false,
+    label: 'Team Members',
+  },
+  {
+    id: 'updatedAt',
+    numeric: false,
+    disablePadding: false,
+    label: 'Updated At',
+  },
+  {
+    id: 'createdAt',
+    numeric: false,
+    disablePadding: false,
+    label: 'Created At',
+  },
+];
+
+function EnhancedTableHead(props) {
+  const { onSelectAllClick, order, orderBy, numSelected, rowCount, onRequestSort } = props;
+  const createSortHandler = (property) => (event) => {
+    onRequestSort(event, property);
+  };
+
+  return (
+    <TableHead>
+      <TableRow>
+        <TableCell padding="checkbox">
+          <Checkbox
+            color="primary"
+            indeterminate={numSelected > 0 && numSelected < rowCount}
+            checked={rowCount > 0 && numSelected === rowCount}
+            onChange={onSelectAllClick}
+            inputProps={{
+              'aria-label': 'select all desserts',
+            }}
+          />
+        </TableCell>
+        {headCells.map((headCell) => (
+          <TableCell
+            key={headCell.id}
+            align={headCell.numeric ? 'right' : 'left'}
+            padding={headCell.disablePadding ? 'none' : 'normal'}
+            sortDirection={orderBy === headCell.id ? order : false}
+          >
+            <TableSortLabel
+              active={orderBy === headCell.id}
+              direction={orderBy === headCell.id ? order : 'asc'}
+              onClick={createSortHandler(headCell.id)}
+            >
+              {headCell.label}
+            </TableSortLabel>
+          </TableCell>
+        ))}
+      </TableRow>
+    </TableHead>
+  );
+}
+
+EnhancedTableHead.propTypes = {
+  numSelected: PropTypes.number.isRequired,
+  onRequestSort: PropTypes.func.isRequired,
+  onSelectAllClick: PropTypes.func.isRequired,
+  order: PropTypes.oneOf(['asc', 'desc']).isRequired,
+  orderBy: PropTypes.string.isRequired,
+  rowCount: PropTypes.number.isRequired,
+};
+
+const EnhancedTableToolbar = (props) => {
```

Comment:
I think we should make this into a separate component to modularize the code and promote maintainability.

Reply:
done